### PR TITLE
Review and Updates for FamilyMemberHistory

### PR DESCRIPTION
**Pull request by RicardoCrespo-ora**
Description
----
In preparation for migrating to Oracle docs, we are reviewing and correcting any inconsistencies, typos, broken links, etc. This PR aims to correct any of the above that needs to be addressed within FamilyMemberHistory and any other information directly referenced by it.

### Capitalization 

**Before**
image

image

**After**

image

image

---

### Fhir docs Modifiers and Extensions hyperlinks

**Before**

image

leads to 

image

which link to nowhere

**After**

image

On click instead lead to appropriate fhir domainresource on modifiers and extensions

---

### Removed redundant Extensions and Modifiers section

**Before**

image

**After**

image

---

### Typo fixes and added missing severity property to extensions section

**Before**

image

**After**

image

---

### Formatting on Parameters table for Search

**Before**

image

**After**

image

### Fixed broken links

**Before**

image
image

**After**
image

---

### Fixed all hyperlinks leading to BackboneElement modifier extension

**Before**

image
image

**After**

image

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [x] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/millennium/r4/clinical/summary/family-member-history.md [hunk 1]
```diff
@@ -28,58 +28,47 @@ The following fields are returned if valued:
 * [Born](https://hl7.org/fhir/r4/familymemberhistory-definitions.html#FamilyMemberHistory.born_x_){:target="_blank"}
 * [Age](https://hl7.org/fhir/r4/familymemberhistory-definitions.html#FamilyMemberHistory.age_x_){:target="_blank"}
 * [Deceased](https://hl7.org/fhir/r4/familymemberhistory-definitions.html#FamilyMemberHistory.deceased_x_]){:target="_blank"}
-  * [Precision Extension](#extensions){:target="_blank"}
-* [Extension](#extensions){:target="_blank"}
-  * [Patient Adopted](#extensions){:target="_blank"}
+  * [Precision Extension](https://hl7.org/fhir/extensibility.html#Extension){:target="_blank"}
```
Comment (amcginn) [inline content/millennium/r4/clinical/summary/family-member-history.md:31]: https://www.hl7.org/fhir/R4/extension-quantity-precision.html

File: content/millennium/r4/clinical/summary/family-member-history.md [hunk 2]
```diff
@@ -28,58 +28,47 @@ The following fields are returned if valued:
...
+* [Extension](https://hl7.org/fhir/extensibility.html#Extension){:target="_blank"}
+  * [Patient Adopted](https://hl7.org/fhir/extensibility.html#Extension){:target="_blank"}
```
Comment (amcginn) [inline content/millennium/r4/clinical/summary/family-member-history.md:33]: I can't tell if this is here
https://hl7.org/fhir/R4/extension-patient-adoptioninfo.html

File: content/millennium/r4/clinical/summary/family-member-history.md [hunk 3]
```diff
@@ -28,58 +28,47 @@ The following fields are returned if valued:
...
 -----------------------------|-----------------------------------------------------------------------------|-----------------------------------------------
 `patient-adopted`            | [`Boolean`](https://hl7.org/fhir/r4/datatypes.html#boolean)                 | Indication of whether a patient is adopted. Only returned when true.
-`precision`                  | [`CodeableConcpet`](https://hl7.org/fhir/r4/datatypes.html#codeableconcept) | Indication of the precision of a given value.
-`condition-course`           | [`CodeableConcpet`](https://hl7.org/fhir/r4/datatypes.html#codeableconcept) | Indication of a condition's progress since diagnosis.
+`precision`                  | [`CodeableConcept`](https://hl7.org/fhir/r4/datatypes.html#codeableconcept) | Indication of the precision of a given value.
+`condition-course`           | [`CodeableConcept`](https://hl7.org/fhir/r4/datatypes.html#codeableconcept) | Indication of a condition's progress since diagnosis.
+`familymemberhistory-severity` | [`CodeableConcept`](https://hl7.org/fhir/r4/datatypes.html#codeableconcept) | A qualification of the seriousness or impact on health of the family member condition.
```
Comment (amcginn) [inline content/millennium/r4/clinical/summary/family-member-history.md:71]: let's align the table column separators

File: content/millennium/r4/clinical/summary/family-member-history.md [hunk 4]
```diff
@@ -28,58 +28,47 @@ The following fields are returned if valued:
...
-`condition-result`           | [`CodeableConcpet`](https://hl7.org/fhir/r4/datatypes.html#codeableconcept) | Indication of the presence (positive) or absence (negative) of a given condition.
-`condition-lifecycle-status` | [`CodeableConcpet`](https://hl7.org/fhir/r4/datatypes.html#codeableconcept) | Indication of whether a condition is active, inactive, resolved, etc.
+`condition-result`           | [`CodeableConcept`](https://hl7.org/fhir/r4/datatypes.html#codeableconcept) | Indication of the presence (positive) or absence (negative) of a given condition.
+`condition-lifecycle-status` | [`CodeableConcept`](https://hl7.org/fhir/r4/datatypes.html#codeableconcept) | Indication of whether a condition is active, inactive, resolved, etc.
 
 
 ### Extensions
 
 ID              | Value\[x] Type                                                    | Description
 -----------------------------|-----------------------------------------------------------------------------|-----------------------------------------------
```
Comment (amcginn) [inline content/millennium/r4/clinical/summary/family-member-history.md:67]: @cldeitrich, I can't tell why these are in the order that they are. do you know why or if we should just make them alphabetical? seems like the HL7 site has them alphabetical(-ish)

File: content/millennium/r4/clinical/summary/family-member-history.md [hunk 5]
```diff
@@ -220,12 +205,12 @@ The common [errors] and [OperationOutcomes] may be returned.
...
 
-    PUT /FamilyMemberHistory/:id
+    PUT /FamilyMemberHistory/:ID
 
 _Implementation Notes_
 
-* Conditions can be removed from a FamilyMemberHistory by setting the 'condition-lifecycle-status' modifierExtension to 'entered-in-error' with system 'http://terminology.hl7.org/CodeSystem/condition-ver-status'. Only existing conditions can be removed; condition id must be provided.
-* An update should only be performed directly after a FamilyMemberHistory is retrieved. If a condition.id is returned on the get operation, it MUST be provided on the subsequent update and must match the condition.id returned from the get.
+* Conditions can be removed from a FamilyMemberHistory by setting the 'condition-lifecycle-status' modifierExtension to 'entered-in-error' with system 'http://terminology.hl7.org/CodeSystem/condition-ver-status'. Only existing conditions can be removed; condition ID must be provided.
+* An update should only be performed directly after a FamilyMemberHistory is retrieved. If a condition.ID is returned on the get operation, it MUST be provided on the subsequent update and must match the condition.ID returned from the get.
```
Comment (amcginn) [inline content/millennium/r4/clinical/summary/family-member-history.md:213]: when we use the resource (`Condition.id`), let's capitalize the resource name

**Review by amcginn — APPROVED** [review]

**Review by cldeitrich — APPROVED** [review]
[PRODOC-203264](https://jira.cerner.com/browse/PRODOC-203264): Approved for style and formatting.

**Review by ashu-bdvt — APPROVED** [review]

**Comment by Surya-Rnsit** [comment]
After Publishing
Screenshot 2023-12-14 at 6 18 09 PM

**Pull request merged**